inspirehep/modules/workflows/tasks/matching.py

```diff
@@ -87,9 +88,12 @@ def fuzzy_match(obj, eng):
     """
     if not current_app.config.get('FEATURE_FLAG_ENABLE_FUZZY_MATCHER'):
         return False
-
+    math_ml_latex_regex = '<mml[^>]*>([^<]+)<\/mml> | <latex[^>]*>([^<]+)<\/latex>'
     fuzzy_match_config = current_app.config['FUZZY_MATCH']
-    matches = dedupe_list(match(obj.data, fuzzy_match_config))
+    obj_data_without_math_ml_latex = deepcopy(obj.data)
```

Comment:
this is wasteful if it's a big record! you're only touching abstracts, so it makes more sense to make a shallow copy, and replace the `abstracts` of that copy only.